common.ts

```diff
@@ -54,6 +54,19 @@ export class Config {
   }
 
   /* Contracts */
+
+  public static L2DepositTrackerAddress(): string {
```

Comment:
Where do these come from when running in CI?

Reply:
they shouldn't be needed, I only put them there if we want to run these load tests against testnet, then it just saves us testnet ETH to not re-deploy the contracts with every test

Reply:
Do you mind adding that as a comment someplace? Either in the `README` or here